source/main/ipc/hardwareWallets/ledger/deviceDetection/deviceTracker.ts

```diff
@@ -12,7 +12,7 @@ export class DeviceTracker {
   }
 
   static getDeviceByPath(path: string): Device {
-    return getDevices().find((d: Device) => d.path === path);
+    return getDevices()?.find((d: Device) => d.path === path);
```

Comment:
I don't think its necessary to add `?` to `getDevices` as it returns an empty array `[]` in case there's no device connected.